### PR TITLE
Relicense under GPLv3+Radicle Linking Exception, require DCO

### PR DESCRIPTION
**Pull request by cloudhead**

### CODE REVIEW

**Comment by cloudhead** [comment]
Is there an easy way to update all license headers?

File: LICENSE [hunk 1]
```diff
@@ -620,56 +649,46 @@ copy of the Program in return for a fee.
 
                      END OF TERMS AND CONDITIONS
 
-            How to Apply These Terms to Your New Programs
+----------------------------------------------------------------------------
 
-  If you develop a new program, and you want it to be of the greatest
-possible use to the public, the best way to achieve this is to make it
-free software which everyone can redistribute and change under these terms.
+                   Developer Certificate of Origin
```
Comment (FintanH) [inline LICENSE:654]: The DCO should be in a separate file

File: .license-header [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // This file is part of radicle-link
 // <https://github.com/radicle-dev/radicle-link>
 //
-// Copyright (C) 2019-2020 The Radicle Team <dev@radicle.xyz>
+// Copyright (C) 2019-2020 The Radicle Foundation <hello@radicle.foundation>
```
Comment (FintanH) [inline .license-header:4]: I believe we're minimising what needs to go in the header. @angedupre said she would get back to me on a shorter copy for it.

**Review by FintanH — CHANGES_REQUESTED** [review]

**Comment by FintanH** [comment]
> Is there an easy way to update all license headers? /cc @FintanH

Not that I know. Maybe a migration `sed` script :smile:

**Comment by kim** [comment]
This one worked for me:

```shell
find . -name '*.rs' -print0 | xargs -0 sed -i -e '/\/\/ This file is part of radicle-link/,/\/\/ along with this program/c\
// Copyright © 2019-2020 The Radicle Foundation <hello@radicle.foundation>\
//\
// This file is part of radicle-link, distributed under the GPLv3 with Radicle\
// Linking Exception. For full terms see the included LICENSE file.'
```

It's not idempotent though

**Comment by kim** [comment]
@cloudhead PTAL

**Review by kim — APPROVED** [review]

**Pull request merged**